src/apis/operations.js

```diff
@@ -10,6 +10,17 @@ operations.resize = (width, height, options = {}) => {
   }
 }
 
+operations.autoRotate = (width, height, rotateDirection, options = {}) => {
```

Comment:
There are no options otherwise than width/height/rotateDirection, correct?
Are all of them required?

Also rotateDirection should be called `rotationDirection`